[IMP] email_marketing: getting started

samueljlieber opened the pull request:
This PR tests the push process of the US doc team. 

## The following process created this PR:

Updated base branch
```
git checkout 16.0
git pull
git log --oneline
```
Created feature branch
```
git checkout -b 16.0-us-docteam-push-sali
```
Stage and commit edits
```
git add .
git commit -m "[IMP] email_marketing: getting started"
```
Push to remote
```
git push origin +16.0-us-docteam-push-sali
```

samueljlieber committed:
[IMP] email_marketing: getting started

samueljlieber commented:
Hi @saliodootest this PR is ready for your review!

saliodootest commented on content/applications/marketing/email_marketing.rst:
```diff
@@ -5,6 +5,8 @@
 Email marketing
 ===============
 
+This is the first change to the document. It a rReally good sentence!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
This sentence is really messed up dude.
```suggestion
This is the first change to the document. It is a really good sentence!
```

saliodootest reviewed (changes requested):
Hi @samueljlieber please find my suggestions below...

samueljlieber commented:
I cannot `git pull --rebase` because there is no tracking information for the current branch.

I can resolve this by first running `git branch --set-upstream-to=origin/16.0-us-docteam-push-sali`

samueljlieber commented:
Squashed @saliodootest's review commit and added a sentence in https://github.com/samueljlieber/dev-documentation/commit/0520dba8c8bea9250ad6b0ef1e1c05415b7e502d.

saliodootest reviewed (approved):
Looks good to me

saliodootest commented on content/applications/marketing/email_marketing.rst:
```diff
@@ -5,6 +5,9 @@
 ===============
 
+This is the first change to the document. It is a really good sentence!
+I added this sentence too because I thought it was cool.
```
BAD SENTENCE I remove
```suggestion
```

saliodootest reviewed (commented):
I like cookies not bad sentences.

samueljlieber commented:
I am going to make a commit locally ignoring your commit here, and try to push up 😈

samueljlieber commented:
I did not have any issues overwriting @saliodootest 's commit with my changes with `git push origin +16.0-us-docteam-push-sali`
